<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(100,100,255,100);</string>
  </property>
  <widget class="QLabel" name="mask_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255,100);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="pm10_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255,100);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="pm25_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>85</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255,100);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="O3_label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255,100);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_show_mask">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255,255,255,100);</string>
   </property>
   <property name="text">
    <string>show mask</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
